https://issues.redhat.com/browse/ACM-21043 (#7977)
open-cluster-management/rhacm-docs: release_notes/acm_deprecate_remove.adoc

| Use fine-grained role-base access control instead.
| See link:../secure_cluster/rbac.adoc#rbac-rhacm[Role-based access control].

| {acm} console
| Importing {ocp-short} 3.11 clusters from the console
| 2.14

> | None

Author: I saw only doc for OCP 4.1. 
```suggestion
| Upgrade to {ocp-short} 4.1 or later. See the link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.15[{ocp-short} documentation].
```

> | The import command for {ocp-short} 3.11 is no longer displayed.

Author: @vishsanghishetty should we recommend that users upgrade their OCP clusters to version 4 and later or just versions later than 3.11?

> | The import command for {ocp-short} 3.11 is no longer displayed.

Author: ```suggestion
| The support for the deprecated custom resource definition `v1beta1` APIs is removed. The `v1beta1` APIs were only needed for {ocp-short} 3.11. The import command for {ocp-short} 3.11 is no longer displayed.
```

> | Upgrade to {ocp-short} 4.1 or later. See the link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.15[{ocp-short} documentation].

Reviewer: @vishsanghishetty @dockerymick we removed 3.11 from our docs and support matrix a while ago. I am confused about why we are talking about 3.11 in the docs for 2.14 after that removal.
 
Is the console just now catching up?

Upgrade to {ocp-short} 4.1 or later. (Missing the version here, I think--but we should prob not say this.)

Author: @swopebe based on the images from the issue. There was a command provided for users who were on OCP 3.11 (this was listed as deprecated from the console), but for 2.14, the command is no longer being displayed

> | Upgrade to {ocp-short} 4.1 or later. See the link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.15[{ocp-short} documentation].

Author: ```suggestion
| Upgrade to a supported version of {ocp-short}. See the link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.15[{ocp-short} documentation].
```

> | The support for the deprecated custom resource definition `v1beta1` APIs is removed. The `v1beta1` APIs were only needed for {ocp-short} 3.11. The import command for {ocp-short} 3.11 is no longer displayed.

Reviewer: ```suggestion
| The support for the deprecated custom resource definition `v1beta1` APIs is removed. The `v1beta1` APIs were only needed for {ocp-short} 3.11. The import command for {ocp-short} 3.11 is no longer displayed and support for this version was previously removed from {acm-short}.
```
